api/store/router.py

```diff
@@ -2,8 +2,8 @@
 
 from store.models.path import PathsDifference
 from store.models.store import StoreRequest, Store
-from store.models.closure import ClosuresDifference, ClosureSize, Closure
-from store.models.package import PackageRequest, PackageMeta, Package, PackageChange, VersionUpdate
+from store.models.package import PackageRequest, PackageMeta, Package, PackageChange, VersionUpdate, Closure, \
+    ClosuresDifference
```

Comment:
```suggestion
from store.models.package import (
    PackageRequest, 
    PackageMeta, 
    Package, 
    PackageChange, 
    VersionUpdate, 
    Closure,
    ClosuresDifference
)
```

надеюсь не наформатировал говна с телефона, но идея наверное понятна